src/containers/PokeList/PokeList.container.js

```diff
@@ -96,8 +87,9 @@ class PokeList extends React.PureComponent {
           renderItem={this.renderPokeItem}
           extraData={loading}
           ListHeaderComponent={this.renderFilter}
-          ListFooterComponent={this.renderLoadMore}
+          ListFooterComponent={this.renderLoadingMore}
           keyExtractor={this.keyExtractor}
+          onEndReached={()=>this.getPokeList()}
```

Comment:
onEndReached={this.getPokeList}

Reply:
This is a open issue in react-native: https://github.com/facebook/react-native/issues/16067

There is a race condition and this implementation is a  possible workaround to avoid a second onEndReached call.

Reply:
This is a open issue in react-native: https://github.com/facebook/react-native/issues/16067

There is a race condition and this implementation is a  possible workaround to avoid a second onEndReached call.

@melihberberolu